site/docs/install-windows.md

```diff
@@ -41,38 +34,63 @@ Alternatively you can:
 *   [Install Bazel from Scoop](#using-scoop)
 *   [Build Bazel from source](install-compile-source.html)
 
-### Step 4 (optional): Configure output directories
+### Step 4: Set up your environment
+
+It's a good idea to rename the Bazel binary to `bazel.exe` and add it to your default paths.
+
+```batch
+set PATH=%PATH%;<path to the Bazel binary>
+```
+
+You can also change your system `PATH` environment variable to make it permanent.
 
-**You can skip this step. Bazel can work without configuring the output directories, and will use
-its default values.**
+### Step 5: Done
 
-By default, Bazel writes to two directories:
+**You have successfully installed Bazel.**
+To check the installation is correct, try to run:
+```batch
+bazel version
+```
+
```

Comment:
"Next, you can check out some best practices, troubleshooting tips and tutorials available below." and remove the list?

Reply:
I moved the best practices section to a windows.md. Here I think it's useful to give a list of links to additional useful infromation.